packages/@react-spectrum/combobox/test/ComboBox.test.js

```diff
@@ -1130,7 +1199,10 @@ describe('ComboBox', function () {
     });
 
     it('supports selectedKey and inputValue (not matching should error)', function () {
-      // TODO by Dan, fill this in when I confirm how the error should work
+      const originalError = console.error;
+      console.error = jest.fn();
+      expect(() => renderComboBox({selectedKey: '2', inputValue: 'Three'})).toThrow();
+      console.error = originalError;
```

Comment:
So that the test doesn't fail due to our `failTestOnConsoleError` in `setupTests.js`. Tried mocking console.error with spyOn but that didn't work so well, open to suggestions

Reply:
it's fine to do this, though use spyOn instead preferably, that way it can't be leaked
also, lets assert something about the log call, otherwise it's not great to block this out

Reply:
that or wrap it in a describe that does the cleanup of the jest.fn()

Reply:
Yeah, I had attempted to use spyOn at first but it kept logging the errors in `failTestOnConsoleError`, fixed it now with a mockImplementation